pkg/cloud/services/networking/floatingip.go

```diff
@@ -179,7 +179,7 @@ func (s *Service) DisassociateFloatingIP(eventObject runtime.Object, ip string)
 }
 
 func (s *Service) waitForFloatingIP(id, target string) error {
-	s.scope.Logger().Info("Waiting for floating IP", "id", id, "targetStatus", target)
+	s.scope.Logger().Info("Waiting for floating IP", "ID", id, "status", target)
```

Comment:
`status` is vague here, it doesn't say if it's the current or expected status. How about `expectedStatus`?

Reply:
The guideline(https://github.com/kubernetes/community/blob/master/contributors/devel/sig-instrumentation/migration-to-structured-logging.md) mentioned "Do not provide additional context for how value is used. Don't use podIP, do use IP." And "Context in name is acceptable to distinguish between values that normally go under same key. For example using both status and oldStatus in log that needs to show the change between statuses."

Reply:
I still disagree, `status` here can be understood both ways.

Reply:
This could go in the log message: "Waiting for floating IP status", and then just "status" in the parameters. I think that would be more explicit.